README.md

```diff
@@ -1,2 +1,7 @@
 # stencil-select
 Stencil's fancy select component
+
+### Previewing stencil-select
+* run `sudo npm install`
```

Comment:
Maybe like this: https://cloud.mobify.com/docs/adaptivejs/install-guide/#/install-grunt/